Updated project ID's

penguwin opened the pull request:
Updated the ID in projects/0001-Minimum_Viable_Microblog.md and
projects/0002-Troll_Proxy.md to fit the filename.

CruzR commented:
I agree that the IDs inside the project descriptions should match the ones in the file name.

The question whether we should count from 0 or 1 remains, i.e. should we change the IDs in those files or rename the files themselves. I'm fine with the suggested solution, so +1 from me.

penguwin committed:
Updated project ID"s

Updated the ID in projects/0001-Minimum_Viable_Microblog.md and
projects/0002-Troll_Proxy.md to fit the filename.
